src/main/kotlin/de/bund/digitalservice/useid/events/EventType.kt

```diff
@@ -0,0 +1,5 @@
+package de.bund.digitalservice.useid.events
+
+enum class EventType(val eventName: String) {
+    SUCCESS("success")
```

Comment:
Do we still need this?

Reply:
I am not sure. Currently we only talked about the happy path of the whole flow. What if errors occur in the eID-Client when reading the data? Do we inform the eService about it? Or do we only send a SSE in case of success?

Reply:
Currently this type is used as event type for the SSE. The enum only makes sense in case we add other types later on.

Reply:
Good point. We should inform the widget also in error cases, so that we can display some error message.

Reply:
I see this as an extension and not necessarily in the scope of this PR. I would error cases when we need them and leave it like that for now. What do you think?

Reply:
Yes, I agree. Didn't want to say anything different, just keeping the thought at the back of my head to handle error messages later :)